### PR TITLE
Add more Bahasa Indonesia book resources

### PR DESCRIPTION
**Pull request by IrvanFza**
## What does this PR do?
Add Resources

### Description 
Add 4 more resources to Bahasa Indonesia books list.

### Why is this valuable (or not)
They are valuable because of very beginner-friendly and easy to understand.

### How do we know it's really free?
They are completely free to download/read.

### For book lists, is it a book?
Yes, they are.

### Checklist:
- [x] Not a duplicate
- [x] Included author(s) if appropriate
- [x] Lists are in alphabetical order
- [x] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-books-id.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
 ### Index
 
+* [Index](#index)
 * [Android](#android)
 * [Emacs](#emacs)
+* [C++](#cplusplus)
 * [Git](#git)
 * [Go](#go)
-* [HTML & CSS](#html-css)
+* [HTML CSS](#html-css)
```
Comment (eshellman) [inline free-programming-books-id.md:9]: check that this links properly

File: free-programming-books-id.md [hunk 2]
```diff
@@ -18,6 +23,10 @@
 * [Tutorial Membuat Aplikasi Galeri Foto Android](https://www.smashwords.com/books/view/533096)
 
 
+### <a id="cplusplus"></a> C++
+
+* [Koding C++ Dengan Qt](https://leanpub.com/koding-cpp-qt)
```
Comment (eshellman) [inline free-programming-books-id.md:28]: we add an access note for leanpub depending on whether the book content can be viewed without registration

**Comment by eshellman** [comment]
also check linter errors

**Comment by IrvanFza** [comment]
What should I do on Alphabetical ordering warning for Index like the following:

```
* [Index](#index)
* [Android](#android)

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Comment by IrvanFza** [comment]
No problem. Great to be part of this community.

**Pull request merged**